src/js/view/controls/controls.js

```diff
@@ -308,12 +331,11 @@ export default class Controls {
 
     userInactive() {
         clearTimeout(this.activeTimeout);
-        this.showing = false;
-        if (this.controlbar) {
-            this.controlbar.closeMenus({
-                type: 'userInactive'
-            });
+        if (this.settingsMenu.visible) {
```

Comment:
`What happens if the volume tooltip menu is open?`
Not sure what you mean by this

`Also, if you're hovering over the control bar, the user-inactive flag does not get added after the timeout.`

This is existing behavior (I verified with build 180).